syntax = "proto3";
package cmdb_extend;

option go_package = "go.easyops.local/contracts/protorepo-models/easyops/model/cmdb_extend";




import "patch_manager_sdk/model/cmdb_extend/app_dependency.proto";

/*
子系统依赖
*/
message SubsystemDependency {
message ConnectSubsystems {
    /*
    系统简称
    */
    string abbreviation = 1 ;
    /*
    模型ID
    */
    string object_id = 2 ;
    /*
    实例ID
    */
    string instance_id = 3 ;
    /*
    名称
    */
    string name = 4 ;
}
        
    
    /*
    系统简称
    */
    string abbreviation = 1 ;
    /*
    系统名称
    */
    string name = 2 ;
    /*
    模型ID
    */
    string object_id = 3 ;
    /*
    实例ID
    */
    string instance_id = 4 ;
    /*
    应用依赖
    */
    repeated cmdb_extend.AppDependency components = 5 ;
    /*
    关联系统
    */
    repeated ConnectSubsystems connect_subsystems = 6 ;
}

